vars/default.yml

```diff
@@ -1,2 +1,5 @@
 ---
-certbot_package: certbot
+certbot_package:
```

Comment:
Yeah, maybe we could switch it to `certbot_packages` for the first variable (a list with one item), and also have `certbot_packages_extra` for additional packages (and not run that task if none are defined).

That's a pattern I use in my PHP role here, and it works great: https://github.com/geerlingguy/ansible-role-php#role-variables